### ISSUE TITLE
from_data wasn't converting nested FlatList

### ISSUE THREAD

**Issue by TensorTom** [klahnakoski/mo-dots#4]
I ran into an issue today with `to_data` not converted nested FlatList to list. I saw you made some commits so I changed my dep to your git's master and it seems to be fixed. Is it on pypi yet? I don't think so.

**Comment by TensorTom** [comment]
Correction: It's still not working. Some nested lists are converted back to list, some aren't.

**Comment by TensorTom** [comment]
I see. It's that some lists are never converted to FlatList to begin with.

**Comment by klahnakoski** [comment]
Yes, this is an unfortunate side effect of using `mo-dots`: The data structures get polluted with `Data` and `FlatList`.  My solution has been to [scrub the datastructure](https://github.com/klahnakoski/mo-json/blob/dev/mo_json/__init__.py#L149) at serialization time.
